test/types/range/unbounded/compareEnumBool.chpl

```diff
@@ -0,0 +1,33 @@
+enum color {red, green, blue};
+use color;
+
+var rgb = red..blue;
+var rdd = red..;
+var ddb = ..blue;
+var empty: range(color, BoundedRangeType.boundedNone) = ..;
```

Comment:
I am surprised that the range `..` is called "empty". How about change the variable name?